### ISSUE TITLE
LVDS screen

### ISSUE THREAD

**Issue by vpeter4** [xbianonpi/xbian-sources-kernel#2]
Can you enable LVDS output for iMX6 kernel? I'm trying myself with not much success.
I appreciate your work.

```
[    0.237188] mxc_sdc_fb fb@0: register mxc display driver ldb
[    0.237308] mxc_sdc_fb fb@0: NO mxc display driver found for ldb!
[    0.237437] mxc_sdc_fb fb@1: register mxc display driver ldb
[    0.237550] mxc_sdc_fb fb@1: NO mxc display driver found for ldb!
```

**Comment by mk01** [comment]
fixed in imx6-4.4.y

**Comment by vpeter4** [comment]
Thank you very, very much.
